### DISCUSSION TITLE
adams_vscode for Adams 2023 & 2024

### DISCUSSION THREAD

**Discussion by mellie0e** [bthornton191/adams_vscode#8] [Q&A]
The ability to debug outside Adams is a real advantage and has enabled us to integrate a lot of Python code into Adams.
Will adams_vscode support Adams 2023 and 2024?

**Comment by bthornton191** [comment]
Glad to hear that you've found it useful @mellie0e. 

I found a fix for 2023 that has consistently worked for me. Please see my comment [here](https://github.com/bthornton191/adams_vscode/issues/6#issuecomment-2192053891) and let me know if this works.  If this ends up being the permemnant fix, I will consider ways to automate it in future versions of adams_vscode.

**Comment by mellie0e** [comment]
Hello Ben,
I installed 2023_2 and 2024_1 this morning
I can debug both versions with 'import threading'.
So, I start working with 2024_1.
Thanks' a lot for your answer.
